### PR TITLE
add a bazel example

### LINKED ISSUE #2: Add a bazel example

### PR DESCRIPTION
**Pull request by jyane**
Fixes #2

### CODE REVIEW

File: examples/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    ],
+    plugins = [
+      ":io_grpc_grpc_java_api_checker_plugin",
+    ],
+)
+
+java_plugin(
+    name = "io_grpc_grpc_java_api_checker_plugin",
+    deps = [
+        "@com_google_guava_guava//jar",
```
Comment (jyane) [inline examples/BUILD.bazel:15]: I didn't think about guava version since this PR, but should we use `guava:19.0` ?

Comment (jyane) [reply examples/BUILD.bazel:15]: Ah... This tool does not require the guava (It is an optional dependency), so the user could use any guava version.

Comment (jyane) [reply examples/BUILD.bazel:15]: I’ve noticed that some dependencies in pom may not be needed, I’ll dig into it.

**Comment by ericgribkoff** [comment]
Can you add instructions for building with bazel to the readme too? Just doing `bazel build` doesn't seem to run the grpc-java-api-checker plugin; are any additional steps (like in https://github.com/google/guava-beta-checker#example) necessary?

**Comment by jyane** [comment]
Thanks!
I've added bazel section to `README.md`.

> Just doing bazel build doesn't seem to run the grpc-java-api-checker plugin; are any additional steps (like in https://github.com/google/guava-beta-checker#example) necessary?

Nope, It works on my local machine, doesn't work on your local machine? If it doesn't work, it means we should add some tests.

```
$ bazel build //...
INFO: Analysed 2 targets (0 packages loaded).
INFO: Found 2 targets...
ERROR: /Users/jyane/workspace/repos/github.com/jyane/grpc-java-api-checker/examples/BUILD.bazel:1:1: Building libexample.jar (1 source file) failed (Exit 1)
src/main/java/com/example/App.java:3: error: [GrpcInternal] @Internal should not be used in application code
import io.grpc.InternalStatus;
              ^
    (see https://github.com/grpc/grpc-java)
src/main/java/com/example/App.java:8: error: [GrpcInternal] @Internal should not be used in application code
    System.out.println(InternalStatus.MESSAGE_KEY);
                                     ^
    (see https://github.com/grpc/grpc-java)
src/main/java/com/example/App.java:8: error: [GrpcInternal] @Internal should not be used in application code
    System.out.println(InternalStatus.MESSAGE_KEY);
                       ^
    (see https://github.com/grpc/grpc-java)
src/main/java/com/example/App.java:9: error: [GrpcExperimentalApi] @ExperimentalApi should not be used in application code
    System.out.println(Grpc.TRANSPORT_ATTR_REMOTE_ADDR);
                           ^
    (see "https://github.com/grpc/grpc-java/issues/1710")
INFO: Elapsed time: 0.272s, Critical Path: 0.14s
FAILED: Build did NOT complete successfully
```

edit: More information on my local machine

```
$ bazel version
...........
Build label: 0.11.1-homebrew
Build target: bazel-out/darwin-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Sun May 31 16:57:39 +50150 (1520426998659)
Build timestamp: 1520426998659
Build timestamp as int: 1520426998659

$ uname -a
Darwin rock.local 17.4.0 Darwin Kernel Version 17.4.0: Sun Dec 17 09:19:54 PST 2017; root:xnu-4570.41.2~1/RELEASE_X86_64 x86_64
```

**Review by ericgribkoff — APPROVED** [review]
LGTM!

The bazel example build works fine, it was just an error on my part earlier

**Comment by jyane** [comment]
Thank you for your review! Merging.

**Pull request merged**